### PR TITLE
[TASK] Revise chapter "XLIFF format"

### PR DESCRIPTION
**Pull request by brotkrueml**
This is a preparation for the new TYPO3 v12.0 feature "Respect attribute approved in XLF files".

Related: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/211
Releases: main, 11.5

### CODE REVIEW

File: Documentation/ApiOverview/Internationalization/XliffFormat.rst [hunk 1]
```diff
@@ -1,118 +1,137 @@
...
-Localizable data are stored in :code:`<trans-unit>` elements. The :code:`<trans-unit>` contains
-a :code:`<source>` element to store the source text and a (non-mandatory) :code:`<target>`
-element to store the translated text.
+Localizable data is stored in :xml:`<trans-unit>` elements. :xml:`<trans-unit>`
+contains a :xml:`<source>` element to store the source text and a
+(non-mandatory) :xml:`<target>` element to store the translated text.
 
-Note that having several :code:`<file>` elements in the same XLIFF document is not
-supported by the TYPO3 CMS Core.
+Remember that the default language is always English, even if you have changed your Typo3 backend to another language, so source-language="en" must be.
```
Comment (linawolf) [inline Documentation/ApiOverview/Internationalization/XliffFormat.rst:22]:
```suggestion
Remember that the default language is always English, even if you have changed 
your TYPO3 backend to another language, it is mandatory to set  :xml:`source-language="en"`.
```

File: Documentation/ApiOverview/Internationalization/XliffFormat.rst [hunk 2]
```diff
@@ -1,118 +1,137 @@
...
 
-Keep in mind that the default language is always considered to be english,
-even when you have changed your typo3 backend to another language, so
-source-language must always be `source-language="en"`.
+Keep in mind that the default language is always English, even when you have
```
Comment (linawolf) [inline Documentation/ApiOverview/Internationalization/XliffFormat.rst:24]: We have this sentence twice now?

File: Documentation/ApiOverview/Internationalization/XliffFormat.rst [hunk 3]
```diff
@@ -1,118 +1,137 @@
...
+changed your TYPO3 Backend to another language, so the source language must
```
Comment (linawolf) [inline Documentation/ApiOverview/Internationalization/XliffFormat.rst:25]:
```suggestion
changed your TYPO3 backend to another language, so the source language must
```

File: Documentation/ApiOverview/Internationalization/XliffFormat.rst [hunk 4]
```diff
@@ -1,118 +1,137 @@
...
+                <trans-unit id="generator" resname="generator">
+                    <source>The "Generator" Meta Tag.</source>
+                </trans-unit>
+            </body>
+        </file>
+    </xliff>
 
-.. note::
+..  tip::
+    The following attributes should be populated properly in order to get the
```
Comment (linawolf) [inline Documentation/ApiOverview/Internationalization/XliffFormat.rst:60]: I wouldn't add this as a tip but as normal text. Admonitions should be short.

File: Documentation/ApiOverview/Internationalization/XliffFormat.rst [hunk 5]
```diff
@@ -1,118 +1,137 @@
...
 
 In the TYPO3 Core, XLIFF files are located in the various system extensions
 as needed and are expected to be located in :file:`Resources/Private/Language`.
 
-In Extbase, the main file (:file:`locallang.xlf`) will be loaded automatically and
-available in the controller and Fluid views without further work needed. Other files will
-need to be referred to explicitly using the :code:`EXT:LLL:extkey/path/to/file:my.label` syntax.
+In :ref:`Extbase <extbase>`, the main file (:file:`locallang.xlf`) will be
+loaded automatically and is available in the controller and Fluid views without
+further work needed. Other files will need to be referred to explicitly using
```
Comment (linawolf) [inline Documentation/ApiOverview/Internationalization/XliffFormat.rst:129]: This sounds a bit bumpy

File: Documentation/ApiOverview/Internationalization/XliffFormat.rst [hunk 6]
```diff
@@ -121,30 +140,29 @@ language files.
 ID naming
 =========
 
-There is no strict rule or guideline in place for defining identifiers
-(the ``id`` attribute).
-Still it is best practice to follow these rules:
+There is no strict rule or guideline for defining identifiers (the :xml:`id`
+attribute). Nevertheless, it is best practice to follow these rules:
```
[the remark this answers is not in the document]
Comment (brotkrueml) [reply Documentation/ApiOverview/Internationalization/XliffFormat.rst:144]: no *strict* rule ;-)

File: Documentation/ApiOverview/Internationalization/XliffFormat.rst (hunk 2 above)
Comment (linawolf) [reply Documentation/ApiOverview/Internationalization/XliffFormat.rst:25]: Please read from line 22 to line 27

**Review by linawolf — APPROVED** [review]

**Pull request merged**